assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/system.properties

```diff
@@ -18,6 +18,12 @@ hideUserHomeFolderOnCreate=false
 # Default folder to open or save file(s) in case user's home folder is hidden
 defaultFolderWhenHomeFolderIsHidden=/public
 
+# Default value to enable migration of Domain objects
+migrate-domains=true
```

Comment:
Not Sure . Are we going to give default value of migrate-domains to true ?

Reply:
yes in this is the file that will be installed at <PBA_DIR>/server/pentaho-server/pentaho-solutions/system/system.properties

Reply:
Also, if the property file does not exists or the value can not be parsed to a Boolean equivalent True or false, I default the default value in PentahoMetadataDomainRepository#DEFAULT_MIGRATE_DOMAINS


see the function  PentahoMetadataDomainRepository#canMigrate(): https://github.com/NJtwentyone/pentaho-platform/commit/9865b39a1889d71cd8c6721f837f51a762dccb0a#diff-d1319d840f943a11cf0cf60e1d517cfba68b0e12e30cc47f2097e819fe71b827R867-R870

Reply:
@himanshu6201 how do you feel about this?